src/Tribe/Updater.php

```diff
@@ -198,10 +185,21 @@ public function flush_rewrites() {
 		add_action( 'wp_loaded', 'flush_rewrite_rules' );
 	}
 
+	/**
+	 * Set the Capabilities for Events and Related Post Types.
+	 *
+	 * @since TBD - change method of calling set_capabilities.
+	 */
 	public function set_capabilities() {
-		$this->capabilities = new Tribe__Events__Capabilities();
-		add_action( 'wp_loaded', array( $this->capabilities, 'set_initial_caps' ) );
-		add_action( 'wp_loaded', array( $this, 'reload_current_user' ), 11, 0 );
+		/* @var Tribe__Events__Capabilities $capabilities */
+		$capabilities = tribe( Tribe__Events__Capabilities::class );
+
+		// We need to set the requirement on update to allow the next page load to trigger
```

Comment:
Use `//` for single line comments only, use `/* ... */` for multi-line comments.